tests/src/unit-alt-string.cpp

```diff
@@ -36,10 +36,33 @@ class alt_string
     alt_string(size_t count, char chr): str_impl(count, chr) {}
     alt_string() = default;
 
-    template <typename...TParams>
-    alt_string& append(TParams&& ...params)
+    alt_string& append(std::size_t count, char ch)
```

Comment:
The template version broke SFINAE-driven feature-detection in `detail/string_concat.hpp`.